src/entity/Courses.java

```diff
@@ -0,0 +1,28 @@
+package entity;
+
+import java.util.List;
+
+public class Courses extends entity.Activity {
```

Comment:
as sugera sa redenumesti clasa in Course la singular, e un best practice ca orice entitate sa fie la singular

Reply:
done